R/correctionsataglance-data.R

```diff
@@ -203,7 +203,7 @@ parseCorrThresholds <- function(reportObject, timezone){
       type <- rep(threshold[['type']], times=nrow(periods))
       startDates <- periods[['startTime']]
       endDates <- periods[['endTime']]
-      value <- periods[['referenceValue']]
+      value <- if(isEmptyOrBlank(periods[['referenceValue']])){NA} else{periods[['referenceValue']]}
```

Comment:
That was my problem. `periods[['referenceValue']]` is a vector, and when I used ifelse, it caused it to only use the first one.
I'm open to changing it to
`if{
} else{
} `
I just thought it looked weird assigning that to a variable.